src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -42,6 +47,12 @@ import { AFFECTED_CLUSTERS_COLUMNS } from '../../AppConstants';
 const ROOT = 'div[id=affected-list-table]';
 const BULK_SELECT = 'clusters-selector';
 const SEARCH_ITEMS = ['ff', 'CUSTOM', 'Foobar', 'Not existing cluster'];
+const VERSION_COMBINATIONS = [
```

Comment:
This is used in the tests, and some tests rely on certain outputs, so we should make sure that we check for that in the data tests. E.g. if we expect that there is at least one cluster with version `'4.18.12'`